index.js

```diff
@@ -23,7 +23,7 @@ var FormComponentMixin = {
   },
 
   handleChange: function handleChange(event) {
-    this.setValue(event.currentTarget.value);
+    this.setValue(event.target.value);
```

Comment:
I didn't see problems with FormsyText (the only other field that uses the method) when I was at 0.2.2. However it looks like you might be splitting out the handleChange functionality with your new PR to fix 0.2.3, so I'm not sure this fix will still apply if/when that happens.